### DISCUSSION TITLE
New minor version with a few PRs integrated

### DISCUSSION THREAD

**Discussion by stefanofornari** [palexdev/MaterialFX#268] [Archived]
Hello Alessandro,
I am wondering if/when you are planning a new minor release including the few open PRs. I know you are working more actively on other projects, but integrating the PRs should not require too much effort, while instead it brings remarkable benefits to the projects using the library because we can clean up code and remove the hacks.

Many thanks!
Ste

**Comment by palexdev** [comment]
I totally agree, I'm gonna merge all the open PRs soon and then release a new version

I finished my work on VirtualizedFX many weeks ago to be honest, and I started the preparations to rewrite MaterialFX with some new technologies and APIs.
December has been an hard month because of university and I'm taking some time off. Don't worry though, I'm planning to start developing again soon
